sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/ScenarioTests/DatabaseAccountOperationsTests.cs

```diff
@@ -0,0 +1,299 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.CosmosDB.Models;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Azure.ResourceManager.CosmosDB.Tests
+{
+    [TestFixture]
+    public class DatabaseAccountOperationsTests : CosmosDBManagementClientBase
+    {
+        private string resourceGroupName;
+        private const string databaseAccountName = "db4937";
+        private const string location = "WEST US";
```

Comment:
We made a location variable in CosmosDBTestUtilities.cs that can be used here by calling CosmosDBTestUtilities.Location; and setting it to be the location everywhere so I don't think you need this private var

Reply:
That location is used when registering a resource group, while each test can use different locations for its operations tests. So it's good to have a location variable here.